libblkid/src/superblocks/stratis.c

```diff
@@ -76,11 +97,15 @@ static int probe_stratis(blkid_probe pr,
 				(buf + SECOND_COPY_OFFSET);
 	}
 
-	blkid_probe_strncpy_uuid(pr, stratis->dev_uuid,
-					sizeof(stratis->dev_uuid));
+	unsigned char formatted_dev_uuid[STRATIS_UUID_STR_LEN];
+	stratis_format_uuid(stratis->dev_uuid, formatted_dev_uuid);
+	blkid_probe_strncpy_uuid(pr, formatted_dev_uuid, sizeof(formatted_dev_uuid));
+
+	unsigned char formatted_pool_uuid[STRATIS_UUID_STR_LEN];
+	stratis_format_uuid(stratis->pool_uuid, formatted_pool_uuid);
```

Comment:
Looks good, but I guess we can use one buffer (e.g `formatted_uuid[STRATIS_UUID_STR_LEN]`) for both UUIDs. It would be also better to define the buffer at begin of the function ;-)